cit/__init__.py

```diff
@@ -14,6 +14,8 @@
 # by modules and controllers
 db = SQLAlchemy(app)
 
+
```

Comment:
what is the purpose of this extra spaces? please read https://www.python.org/dev/peps/pep-0008/